renderer/src/components/JiraConnection/JiraConnection.tsx

```diff
@@ -16,7 +16,7 @@ const JiraConnection = () => {
     const online = await isOnline();
 
     if (online) {
-      global.ipcRenderer.send(IPC_MAIN_CHANNELS.JIRA_LOGIN);
+      global.ipcRenderer.send("open-child-window", "jira");
```

Comment:
Could you please move these two strings into a const? electron-src/helpers/constants.ts